driver/command/command.go

```diff
@@ -101,5 +119,62 @@ func (d *Driver) exec(op *driver.Operation) (driver.OperationResult, error) {
 		return driver.OperationResult{}, fmt.Errorf("Start of driver (%s) failed: %v", d.Name, err)
 	}
 
-	return driver.OperationResult{}, cmd.Wait()
+	if err = cmd.Wait(); err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed executing bundle: %v", d.Name, err)
+	}
+
+	result, err := d.getOperationResult(op)
+	if err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed getting operation result: %v", d.Name, err)
+	}
+
+	return result, nil
+}
+func (d *Driver) getOperationResult(op *driver.Operation) (driver.OperationResult, error) {
+	opResult := driver.OperationResult{
+		Outputs: map[string]string{},
+	}
+
+	if len(op.Outputs) == 0 {
+		return opResult, nil
+	}
+
+	for _, item := range op.Outputs {
+		// Check if a value is required for this output and get the default value if available
+		valueRequired, defaultValue, err := checkIfOutputValueRequired(item, op, opResult)
+		if err != nil {
+			return opResult, fmt.Errorf("Command driver (%s) failed checking if output value required for item: %s Error: %v", d.Name, item, err)
+		}
+
+		fileName := path.Join(d.outputDirName, item)
+		_, err = os.Stat(fileName)
+		if err != nil {
+			if os.IsNotExist(err) {
+				// Set a default value if available and required otherwise fail
+				if valueRequired {
+					return opResult, fmt.Errorf("Command driver (%s) failed for item: %s no output value found and no default value set", d.Name, item)
+				}
+				opResult.Outputs[item] = defaultValue
+				continue
+			}
+			return opResult, fmt.Errorf("Command driver (%s) failed checking for output file: %s Error: %v", d.Name, item, err)
+		}
+
+		contents, err := ioutil.ReadFile(fileName)
+		if err != nil {
+			return opResult, fmt.Errorf("Command driver (%s) failed reading output file: %s Error: %v", d.Name, item, err)
+		}
+
+		opResult.Outputs[item] = string(contents)
+	}
+
+	return opResult, nil
+}
+func checkIfOutputValueRequired(item string, op *driver.Operation, opResult driver.OperationResult) (bool, string, error) {
+	// TODO Check if the output is required by the action and write test to validate (requires update to op definition in https://github.com/deislabs/cnab-go/pull/129 to be merged )
```

Comment:
I wonder if we shouldn't mark this PR as `[WIP]` as it depends on another PR. I feel a bit uncomfortable merging the code as is, with this unimplemented feature.

Reply:
Yeah buts its unimplemented everywhere until #129 is merged, once that is done its going to be straightforward to implement here, alternative I can remove this placeholder function

Reply:
This is implemented now